# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-rustc-t2.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-103.0/build/moz.configure/rust.configure.vanilla	2022-08-09 12:08:08.707934940 +0200
+++ firefox-103.0/build/moz.configure/rust.configure	2022-08-09 12:18:52.980909250 +0200
@@ -394,7 +394,7 @@
         if len(narrowed) == 1:
             return narrowed[0].rust_target
 
-        return None
+        return candidates[1].rust_target
 
     rustc_target = find_candidate(candidates)
 
